pkg/beaconfqdn/mongodb.go

```diff
@@ -117,9 +122,31 @@ func (r *repo) Upsert(hostnameMap map[string]*hostname.Input, minTimestamp, maxT
 		writerWorker.start()
 	}
 
+	s := spinner.New(spinner.CharSets[36], 200*time.Millisecond)
+	s.Prefix = "\t[-] Gathering FQDNs for Beacon Analysis ...\t"
+	s.Start()
```

Comment:
Something seems to be just a bit off with moving from this spinner to the next spinner. I tested twice and see something like this each time:

![image](https://user-images.githubusercontent.com/11990044/130503378-574ea5c0-b6f7-4de4-bac7-aafd64fce849.png)

Reply:
I cannot reproduce this issue in my terminal. In my terminal, the second spinner overwrites the first one.
I'll try adding a `println` call after `s.Stop()`, but I'll need you to let me know whether it helped or not.

Reply:
It could be an ESXi thing. I've had other weird terminal things happen too that nobody else experienced.